app/lib/frontend/templates/views/shared/toc.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../../../dom/dom.dart' as d;
+
+/// Renders Table of Contents from [nodes].
+///
+/// The hiearchy of the sections is encoded in [TocNode.children].
+d.Node renderToc(Iterable<TocNode> nodes) {
+  return d.div(
+    classes: ['pub-toc-container'],
+    child: d.div(
+      classes: ['pub-toc'],
+      children: [
+        d.div(child: d.b(text: 'Contents')),
```

Comment:
Open question. Should we (specifically for the admin page) call this 
```suggestion
        d.div(child: d.b(text: 'Sections')),
```

Reply:
I think it is fine to call it `Sections` on any page really. It is not that we can guarantee that a random README actually has a content, only the section headers.